tests/__snapshots__/test_amber_data_serializer.ambr

```diff
@@ -1,24 +0,0 @@
-# name: test_produces_valid_object_types[0]
-  "<class 'str'>"
----
-# name: test_produces_valid_object_types[1]
-  "<class 'int'>"
----
-# name: test_produces_valid_object_types[2]
-  "<class 'tuple'>"
----
-# name: test_produces_valid_object_types[3]
```

Comment:
we should make sure we've tested each type in the remaining test files. e.g. booleans

Reply:
Sure